dlt/common/destination/reference.py

```diff
@@ -518,6 +519,22 @@ def _raw_capabilities(self) -> DestinationCapabilitiesContext:
         """Returns raw capabilities, before being adjusted with naming convention and config"""
         ...
 
+    @property
+    def destination_info(self) -> Dict[str, Any]:
+        """Return the destination info as a dict.
+
+        Returns:
+            Dict[str, Any]: Destination info.
+        """
+        conf = self.configuration(self.spec())
```

Comment:
pls pass `accept_partial=True` here so we can get basic config info without ie. credentials

Reply:
Done